packages/api-sdk/src/hooks/hooks.ts

```diff
@@ -2,6 +2,7 @@
  * Code generated by Speakeasy (https://speakeasy.com). DO NOT EDIT.
  */
 import { RequestInput } from "../lib/http.js";
+import { initHooks } from "./registration.js";
```

Comment:
i think this is legacy code from the registration endpoint? my most recent `api-sdk` speakeasy generation removed it, and this looks to be adding it back in

Reply:
actually idk. not sure what the diff is from...i think we're using the same speakeasy CLI version.